src/Database/Table/Selection.php

```diff
@@ -334,6 +334,33 @@ public function where($condition, ...$params)
 		return $this;
 	}
 
+	/**
+	 * Adds where condition using the OR operator between parameters.
+	 * More calls appends with AND.
+	 * @param  array ['column1' => 1, 'column2 > ?' => 2, 'full condition']
+	 * @return self
+	 */
+	public function whereOr(array $parameters)
+	{
+		if (count($parameters) < 2) {
+			return $this->where($parameters);
+		}
+		$columns = [];
+		$values = [];
+		foreach ($parameters as $key => $val) {
+			if (is_int($key)) { // whereOr(['full condition'])
+				$columns[] = $val;
+			} else if (strpos($key, "?") === FALSE) { // whereOr(['column1' => 1])
+				$columns[] = $key . " ?";
+				$values[] = $val;
+			} else { // whereOr(['column1 > ?' => 1])
+				$columns[] = $key;
+				$values[] = $val;
```

Comment:
I think, this should also support multiple parameters, e.g.
`['SOME_FN(column, ?, ?)' => [1, 2]]`

maybe this way?

```php
$values = array_merge($values, substr_count($key, '?') > 1 ? $val : [$val]);
```